scripts/microbe_stage/microbe_replacement.lua

```diff
@@ -64,5 +54,20 @@ function MicrobeReplacementSystem:activate()
     end
 end
 
+--Faux-latin name generation routine (Move to own file eventually?)
+function MicrobeReplacementSystem:generateSpeciesName()
+
+    --prefix,cofix,suffix list
+    speciesNamePrefix = {' Ce', ' Ar',' Sp', ' Th',' Co', ' So', ' Pu', ' Cr', ' Cy', ' Gr', ' Re', ' Ty', ' Tr', ' Ac',' Pr' }
+    speciesNameCofix = { 'nan', 'mo', 'na', 'yt', 'yn', 'il', 'li','le', 'op', 'un', 'rive','ec', 'ro','lar','im' }
+    speciesNameSuffix = { 'pien', 'olera', 'rius', 'nien', 'ster', 'ilia', 'canus', 'tus', 'cys','ium','um'}
```

Comment:
General development best practices (and also personal preference): variables that don't change over the execution of a function should be seen as a constant. Or for a "technical" reason, these table are instantiated every time the function run, so less memory used when stored in a constant (this is so NOT a good argument with the size of the tables though).

Lastly, if somebody wants to add more pre/co/suffix, they won't have to search the entire file since they'll be in a constant in the top of the file.